__tests__/commands/docs/upload.test.ts

```diff
@@ -24,6 +24,9 @@ describe('rdme docs upload', () => {
       categories: {},
       parentPages: {},
     });
+    getAPIv2Mock({ authorization }).get('/projects/me').reply(200, {
```

Comment:
these tests are great, ty! could you also add a test case for if this initial `/projects/me` request fails? i'll DM you an example error response that you can pull from shortly